api/lib/application/organizations/index.js

```diff
@@ -526,6 +526,7 @@ exports.register = async (server) => {
           }),
           query: Joi.object({
             division: Joi.string().required(),
+            isFrenchDomainExtension: Joi.boolean(),
```

Comment:
Si possible, toujours renseigner la valeur envoyée par le front 
"Better explicit than implicit"
```suggestion
            isFrenchDomainExtension: Joi.boolean().required(),
```

Reply:
Done